ci-operator/step-registry/assisted/baremetal/operator/ztp/assisted-baremetal-operator-ztp-workflow.yaml

```diff
@@ -10,6 +10,7 @@ workflow:
     - ref: assisted-baremetal-operator-gather
     - chain: baremetalds-ipi-post
     env:
+      PACKET_PLAN: "c3.medium.x86"
```

Comment:
did you do the calculation of how it adds up with the most resourceful job? (in terms of disks + CPU + MEM)
I think ``e2e-ai-operator-ztp-3masters`` should be the most resourceful, taking into account those nodes disks and the LSO disks

Reply:
yes I would like to pj-rehease all the ztp jobs to see the ones who don't pass